synthesizer/src/process/stack/fee/mod.rs

```diff
@@ -76,9 +78,50 @@ impl<N: Network> Fee<N> {
         self.global_state_root
     }
 
-    /// Returns the inclusion proof.
-    pub const fn inclusion_proof(&self) -> Option<&Proof<N>> {
-        self.inclusion_proof.as_ref()
+    /// Returns whether the fee proves inclusion
+    pub fn proves_inclusion(&self) -> bool {
+        if self.proof.is_some() { self.proof.as_ref().unwrap().proves_inclusion() } else { false }
```

Comment:
`if let Some(proof) = &self.proof (...)` would be tidier

Reply:
Done: 1ff86a0db